cernopendata_client/searcher.py

```diff
@@ -192,20 +192,34 @@ def get_files_list(
     :return: List of files list
     :rtype: list
     """
-    if verbose:
-        files_list = [
-            (file["uri"], file["size"], file["checksum"])
-            for file in record_json["metadata"]["files"]
-        ]
-        if protocol == "http":
-            files_list = [
-                (file_[0].replace(SERVER_ROOT_URI, server), file_[1], file_[2])
-                for file_ in files_list
-            ]
-    else:
-        files_list = [file["uri"] for file in record_json["metadata"]["files"]]
-        if expand:
-            # let's unwind file indexes
+    files_list = []
+    for file_ in record_json["metadata"]["files"]:
+        if verbose:
```

Comment:
`verbose` is an output-level variable, so it should not really influence how we gather file information based on `expand`.
What about collecting always everything, i.e. uri/size/checksum triad, and use `verbose` only at the end to either output them or not? In this way the info-collecting part (using `expand`) and ouput-printing part (using `verbose`) will be more nicely separated.

Reply:
okay, sounds good 👍